modules/users/server/models/user.server.model.js

```diff
@@ -300,6 +300,11 @@ var UserSchema = new Schema({
   pushRegistration: {
     type: [UserPushRegistrationSchema],
     default: []
+  },
+  acquisitionStory: {
+    type: String,
+    trim: true,
+    default: ''
```

Comment:
[set: setPlainTextField](https://github.com/Trustroots/trustroots/pull/970/files/9fd92117593174b6cbb7ce8a7f841fa3a153cf33..eec96a3a25744b20bd77bc75cce432bf9418c1b4#diff-240057bbb9109acf3b1745506df4b781L134) can be interesting for us here.

Reply:
Yep, DRYer and more consistent.